webpack.config.js

```diff
@@ -43,6 +45,10 @@ const shouldCheckForCycles = process.env.CHECK_CYCLES === 'true';
 const codeSplit = config.isEnabled( 'code-splitting' );
 const isCalypsoClient = process.env.CALYPSO_CLIENT === 'true';
 
+const browserslistEnvironment = process.env.BROWSERSLIST_ENV || 'evergreen';
```

Comment:
Could we have this be set to `defaults` if `! isCalypsoClient`?

Otherwise, SDK builder will always build `evergreen` and Jetpack and wpcom aren't ready for that just yet. :-)

If this can be a CLI argument in [`bin/sdk-cli.js`](https://github.com/Automattic/wp-calypso/blob/8dfcfced52ac061ff6cde1d25905da455cb1e5bb/bin/sdk-cli.js) that defaults to `defaults`, that would be ace!

Reply:
Seems like the WebPack build is doing more than I thought :-/ Done!